### PR TITLE
feat: add growing card

### PR DESCRIPTION
**Pull request by ralucacril**
Screenshot 2020-08-13 at 13 07 54
Screenshot 2020-08-13 at 13 07 46

### CODE REVIEW

File: src/components/card/growing.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+interface Props {
+  renderImage: () => ReactNode
+  children: ReactNode
+  noMarginsOnMobile?: boolean
+}
+
+export const GrowingCard: FC<Props> = ({
+  renderImage,
+  children,
+  noMarginsOnMobile,
```
Comment (lkappeler) [inline src/components/card/growing.tsx:13]: It's not really related to margins, isn't it? More like `roundedCornersOnMobile`

**Comment by masone** [comment]
as discussed - we don't think that we can leverage this to make the dealer cards happen. ergo, this would be only used in listings. I would recommend not extracting it in that case to spare us from unnecessary maintenance overhead. only extract it if you see that we're sharing this component across projects

**Comment by ralucacril** [comment]
@masone yap, i will not extract this.

File: src/components/card/growing.tsx (hunk 1 above)
Comment (masone) [reply src/components/card/growing.tsx:13]: I did not catch that, but now that you point at it - the name here seems overly specific. can we generalize it more? what would be a "semantic" approach? we have for example a `fullWidth` property on the `AlertMessage` https://github.com/carforyou/carforyou-components-pkg/blob/136ab6c03c84012f37c24568b09a566f80904819/src/components/alertMessage.tsx#L38

if `fullWidth` -> non-rounded corners
else -> rounded corners

Comment (masone) [reply src/components/card/growing.tsx:13]: oh my. we closed this PR already 🙈

**Pull request closed without merging**